.github/workflows/library-dragonfly1.14.yaml

```diff
@@ -58,9 +57,25 @@ jobs:
         path: ${{ github.workspace }}/.kraftkit/oci/digests
         if-no-files-found: error
 
+  test:
+    needs: [build]
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Run Dragonfly with QEMU and test with redis-cli
+      uses: unikraft/kraftkit@staging
+      working-directory: library/dragonfly/1.14
+      with:
+        run: |
+          set -xe
+          kraft run --rm -p 6379:6379 --plat qemu --arch x86_64 -M 256M unikraft.org/dragonfly:1.14 &
```

Comment:
Are you sure this runs the image built above and not the old/already pushed image?

You might have to use `index.unikraft.io/unikraft.org/dragonfly:1.14` & do some tricks like adding stuff like `--no-pull` to the command to work.

Also, to do this you will have to use a `download-artifact` action after `checkout`

You can try locally first: build the dragonfly image and then try to use it from another directory.